docs/source/eks.md

```diff
@@ -110,10 +110,14 @@ kubectl apply -f node-setup-daemonset.yaml
 
 Now you can follow the [generic guide](generic.md) to launch your Scylla cluster in a highly performant environment.
 
-#### Deleting a EKS cluster
+#### Accessing the database
 
-Once you are done with your experiments delete your cluster using following command:
+Instructions on how to access the database can also be found in the [generic guide](generic.md).
```

Comment:
any chance for direct link to subsection?

Reply:
Tried the direct link and it didn't work locally so I kept it that way